core/arch/arm/plat-marvell/conf.mk

```diff
@@ -3,13 +3,27 @@ PLATFORM_FLAVOR ?= armada7k8k
 ifeq ($(PLATFORM_FLAVOR),armada7k8k)
 include core/arch/arm/cpu/cortex-armv8-0.mk
 CFG_TEE_CORE_NB_CORE = 4
+CFG_TZDRAM_START = 0x04400000
```

Comment:
@jforissier: here we should rather use `$(call force,CFG_TZDRAM_START,0x04400000)`, no?

Reply:
If the address is a fixed value, such as defined by the hardware, then yes.
If it may be changed (for instance a prior firmware step configures some DRAM range as secure via a memory firewall, and that range is configurable -- like on HiKey), then `?=` is fine.

Reply:
@kevin-peng-hao, do you want TZDRAM and SHMEM to be configurable, or hard-coded to current values?

Reply:
Let it be hard-coded for now. Thanks.